### PR TITLE
Expose nginx status to prometheus in Kubernetes (helm)

### PR DESCRIPTION
**Pull request by uncycler**
This PR add the possibility to expose nginx status to prometheus using the nginx prometheus exporter. For doing so, I had to disable metrics basic auth for localhost to allow the exporter to access the nginx status endpoint.

### CODE REVIEW

File: KUBERNETES.md [hunk 1]
```diff
@@ -308,6 +308,10 @@ However, that doesn't work and I haven't found out why. In a production
 environment, a redundant PostgreSQL cluster is the better option. As it uses
 statefulsets that are kept by default, the problem doesn't exist there.
 
+### Prometheus metrics
+
+It's possible to enable Nginx prometheus exporter by setting `--set monitoring.enabled=true` and `--set monitoring.prometheus.enabled=true`. This is add the Nginx exporter sidecar and and the standard pod annotations to django deployment.
```
Comment (madchap) [inline KUBERNETES.md:313]: This is add --> This adds

and and --> and

**Review by madchap — APPROVED** [review]
lgtm. Thanks for the PR.

**Review by alles-klar — APPROVED** [review]

**Comment by alles-klar** [comment]
@uncycler please set target branch to dev

**Comment by valentijnscholten** [comment]
@alles-klar I've changed the base branch of the PR and still looks good. Can you merge if you agree?

**Pull request merged**